readme.md

```diff
@@ -4,34 +4,34 @@
 
 # Task description
 
-Implement a function accepting 2 arguments: `state` and `transforms`.
-The function should return an array of the same length as `transforms`.
+Implement a function accepting 2 arguments: `state` and `actions`.
+The function should return an array of the same length as `actions`.
 The array should contain all previous versions of `state`.
```

Comment:
Write a function `transformStateWithClones` that take a `state` object and an `actions` array and return an array of the same length as `actions` containing all previous versions of the `state`.